src/index.js

```diff
@@ -10,6 +13,7 @@ app.use(express.urlencoded({ extended: false }));
 app.use(express.json());
 
 const isProduction = process.env.NODE_ENV === 'production';
+app.use('/api/v1/users', usersRoute);
```

Comment:
Why is this here? This supposed to be handled in the route directory. Refer to @oyewoas's task to refactor this routing and put the logic where it belongs